### PR TITLE
victim-perpetrator?

### PR DESCRIPTION
**Pull request by adelaidaa**

### CODE REVIEW

File: src/main/resources/db/migration/V1_5__pre_sentence_assessment.sql [hunk 1]
```diff
@@ -160,7 +160,8 @@ VALUES ('1ea17930-d467-44c0-9e72-517747618acb', 'ui3.1', '2020-11-30 14:50:00',
 ('3211a668-8878-4e88-8457-8250bfe65aea', '80.1', '2020-11-30 14:50:00', null, 'radio', 'f95ff568-cfa0-4dec-b5f4-f82c950b24d4', 'Is the person unemployed or will be unemployed upon release?', null, null, null),
 ('1970ba5e-91cb-4ad3-9f04-64d5b5b7157b', '86.1', '2020-11-30 14:50:00', null, 'radio', 'd03940ce-5f84-4ec1-af45-ab2957d09402', 'What is the person''s current relationship with their partner?', null, null, null),
 ('f04dd882-0a0d-49f5-9736-91eeadbff9e7', '87.1', '2020-11-30 14:50:00', null, 'radio', '887f4528-06d1-4247-8bc3-5e679222baa6', 'Is there evidence that the individual is a perpetrator of domestic abuse?', null, null, null),
-('fb98f8b1-58ed-4aac-85b1-404f84270b95', '88.1', '2020-11-30 14:50:00', null, 'radio', 'f01257fa-c05b-40fc-a134-1fcf6809e9f9', 'Victim or perpetrator', null, null, null),
+('fb98f8b1-58ed-4aac-85b1-404f84270b95', '88.1', '2020-11-30 14:50:00', null, 'checkbox', 'f01257fa-c05b-40fc-a134-1fcf6809e9f9', 'Victim', null, null, null),
```
Comment (LucasCairns) [inline src/main/resources/db/migration/V1_5__pre_sentence_assessment.sql:163]: These are linked to the Victim/Perpetrator AnswerSchema - does this mean that both these checkbox groups will display bother answers?

Comment (adelaidaa) [reply src/main/resources/db/migration/V1_5__pre_sentence_assessment.sql:163]: no idea atm, trying to figure it out

Comment (LucasCairns) [reply src/main/resources/db/migration/V1_5__pre_sentence_assessment.sql:163]: Let's merge and see in the management tool :)

**Review by LucasCairns — APPROVED** [review]
LGTM 👍

**Review by SWhitlam — APPROVED** [review]

**Pull request merged**